api/server.js

```diff
@@ -0,0 +1,167 @@
+// imports
+const express = require("express");
+const morgan = require("morgan");
+const helmet = require("helmet");
+const cors = require("cors");
+
+// DB import
+const userDB = require("../data/helpers/userDb");
+const postDB = require("../data/helpers/postDb");
+
+// create server instance
+const server = express();
+
+// Custom Middleware
+
+function checkAllCaps(req, res, next){
+    const name = req.body.name;
+    const arr = name.split(" ");
+    let result;
+    for(let i = 0; i < arr.length; i++){
+        if(arr[i][0] !== arr[i][0].toUpperCase()){
```

Comment:
It might be beneficial to have `arr[i][0]` in a variable since you use it twice and it might make it slightly more readable:
```javascript
const firstChar = arr[i][0];
if(firstChar !== firstChar.toUpperCase()) {
```